cmd/vc-rest/startcmd/start.go

```diff
@@ -329,9 +388,10 @@ func startEdgeService(parameters *vcRestParameters, srv server) error {
 		externalHostURL = parameters.hostURLExternal
 	}
 
-	vcService, err := vc.New(&operation.Config{StoreProvider: storeProvider,
-		EDVClient: edv.New(parameters.edvURL, edv.WithTLSConfig(tlsConfig)),
-		KMS:       kms, VDRI: vdri, HostURL: externalHostURL, Mode: parameters.mode, Domain: parameters.blocDomain,
+	vcService, err := vc.New(&operation.Config{StoreProvider: edgeServiceProvs.provider,
+		KMSSecretsProvider: edgeServiceProvs.kmsSecretsProvider,
```

Comment:
create localkms and pass it like legacy kms

Reply:
done